scripts/gen_tee_bin.py

```diff
@@ -72,15 +72,20 @@ def get_symbol(elffile, name):
                          if isinstance(s, SymbolTableSection)]
         for section in symbol_tables:
             for symbol in section.iter_symbols():
+                # .name might be a byte array or a string, we want a string
```

Comment:
How about adding a helper function for this? `get_symbol_name()` for instance.

Reply:
Sure.